src/Board.py

```diff
@@ -118,12 +136,12 @@ def get_indirectly_adj(
                 adjacent_tile_list.append(tile)
         return adjacent_tile_list
 
-    def check_adjacency(self, start, end, shipping_limit) -> str:
+    def check_adjacency(self, start, end, shipping_limit) -> AdjacencyType:
         if end in self.get_directly_adj(start, None):
-            return "Direct"
+            return AdjacencyType.DIRECT
         if end in self.get_indirectly_adj(start, None, shipping_limit):
-            return "Indirect"
-        return "Not"
+            return AdjacencyType.Indirect
```

Comment:
Thanks for adding the enum here! Just one thing, can we please keep these consistently uppercase? (uppercase is traditionally used for constants)